install.sh

```diff
@@ -53,6 +53,8 @@ CATKIN_DIR=~/mil_ws
 MIL_CONFIG_DIR=~/.mil
 BASHRC_FILE=~/.bashrc
 MILRC_FILE=$MIL_CONFIG_DIR/milrc
+BVTSDK_DIR=$MIL_CONFIG_DIR
+BVTSDK_DIR_COMPLETE=$BVTSDK_DIR/bvtsdk
```

Comment:
I like it being it's own variable so it's easy to change one line (56) if we decide to put BVTSDK somewhere else like .local or something.